src/Plate/index.tsx

```diff
@@ -0,0 +1,110 @@
+import { range, uniq } from 'lodash';
+import React, { ReactNode } from 'react';
+
+import { MLL_THEME } from '../theme';
+
+export type PlateProps = {
+  data?: Array<{
```

Comment:
```suggestion
  data: Array<{
```

Makes little sense to not pass `data`.